instruments/agilent/agilent34410a.py

```diff
@@ -162,7 +162,7 @@ def read_last_data(self):
         }
 
         if data == '9.91000000E+37':
-            return int(data)
+            return float(data)
```

Comment:
Previous version returned:
```python
ValueError: invalid literal for int() with base 10: '9.91000000E+37'
```
Furthermore, this method otherwise returns a `float`. Changing the return in this special case to a float as well.